<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fzqjGetPicSizeClass</class>
 <widget class="QWidget" name="fzqjGetPicSizeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="8,1,1">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,3">
     <item>
      <widget class="RectLabel" name="windShowPicVi">
       <property name="text">
        <string>文字1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RectLabel" name="windShowPicIr">
       <property name="text">
        <string>文字2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,2,13,1,1">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>焦距(毫米)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editFocusValue"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>距离(米)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editDisValue"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCount">
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImageRefresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,3,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>宽度(毫米)：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editWidth"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>高度(毫米)：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editHeight"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>面积(米)：</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editSize"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RectLabel</class>
   <extends>QLabel</extends>
   <header>RwriteLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
